src/Sylius/Behat/Page/Admin/CatalogPromotion/CreatePage.php

```diff
@@ -13,13 +13,39 @@
 
 namespace Sylius\Behat\Page\Admin\CatalogPromotion;
 
+use Behat\Mink\Element\NodeElement;
 use Sylius\Behat\Behaviour\SpecifiesItsCode;
 use Sylius\Behat\Page\Admin\Crud\CreatePage as BaseCreatePage;
+use Sylius\Behat\Service\AutocompleteHelper;
+use Webmozart\Assert\Assert;
 
 class CreatePage extends BaseCreatePage implements CreatePageInterface
 {
     use SpecifiesItsCode;
 
+    public function addProductCodeToRuleCongiguration(?string $productName): void
```

Comment:
I guess that there is already a method in `FormElement` to specify variants for rule configuration